src/App.js

```diff
@@ -4,9 +4,21 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
 
   render() {
+    let copy = [...moviesFromServer];
+
+    if (this.state.query.length > 0) {
+      const filterQuery = this.state.query.toLowerCase();
+
+      copy = copy
+        .filter(movie => (movie.title.toLowerCase().indexOf(filterQuery) >= 0)
+      || (movie.description.toLowerCase().indexOf(filterQuery) >= 0));
+    }
```

Comment:
```suggestion
    const { query } = this.state;
    const visibleMovies = moviesFromServer.filter(
    ({ title, description }) => title.toLowerCase().includes(query)
      || description.toLowerCase().includes(query);
    }
```
Учитывая, что метод includes с пустой строкой всегда вернет true, при поиске по любой строке, можно избавиться от проверки if и немного облегчить код

Reply:
Illia is right. Better to use `includes` here and get rid of unnecessary `if`